src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+function createDiv(divClass) {
+  const div = document.createElement('div');
+
+  div.className = divClass;
+
+  if (divClass === 'message') {
+    div.textContent = 'Promise was resolved!';
+  } else {
+    div.textContent = 'Promise was rejected!';
+  }
+
+  return div;
+}
+
+const promise1 = new Promise(resolve => resolve(createDiv('message')));
```

Comment:
Here the promise is resolved immediately, on click you just take the resolved value and use it. Try to resolve this promise only after clicking on the logo.